aubreylib/resource.py

```diff
@@ -1,16 +1,16 @@
 import os
 import re
-import StringIO
+from io import BytesIO
 import datetime
-import urllib2
+import urllib.request
+import urllib.error
+import urllib.parse
```

Comment:
Doesn't look like we use `urllib.error` or `urllib.parse`. Need to look through the same imports in other modules.